# DVL Dead Reckoning message
std_msgs/Header header
float64 time
geometry_msgs/Vector3 position
float64 pos_std
float64 roll
float64 pitch
float64 yaw
string type
int64 status
string format
